implementations/code_df/new_contributors_of_commits.py

```diff
@@ -0,0 +1,147 @@
+import pandas as pd
+from datetime import datetime
+import utils
+import conditions
+from commit import Commit
+
+
+class NewContributorsOfCommits(Commit):
+    """
+    Class for New contributors of commits.
+
+    :param items: A list of dictionaries, each element a line from the
+        JSON file with Perceval data
+    :param date_range: A tuple which represents the start and end date of
+        interest
+    :param issrccode_obj: An object of IsSourceCode, used to determine
+        what comprises source code.
+    """
+
+    def compute(self, check_range=(None, None)):
+        """
+        Count number of new committers who committed between the two dates
+        of check range.
+
+        :param check_range: A tuple which represents the start and end date
+            when new committers will be considered
+
+        :returns count: the number of new committers who committed
+            between the dates of check_range
+        """
+
+        self.check_since, self.check_until = check_range
+
+        if self.check_since is None:
+            self.check_since = self.since
+
+        if self.check_until is None:
+            self.check_until = self.until
+
+        df = self.df[self.df['created_date'] < self.check_since]
+
+        committers_before_check_since = set(df['author'].tolist())
+        df_new_committers = self.df['author'][
+                    (self.check_since <= self.df['created_date'])
+                    & (self.check_until >= self.df['created_date'])
+                    & (~self.df['author']
+                        .isin(
+                            list(committers_before_check_since)))
+                ]
+        count_new_committers = len(df_new_committers.unique())
+        return count_new_committers
+
+    def compute_timeseries(self, period='month'):
+        """
```

Comment:
Can we use the current `time_series` in your pull request, instead of this implementation?

Reply:
Very sorry. I made the changes and made a commit for the notebook, forgetting to add this file. 
Please have a look now with the latest changes. Regarding your queries of why I am renaming functions, I have addressed them in the Code Changes Git pull request (#176)

Reply:
Very sorry. I made the changes and made a commit for the notebook, forgetting to add the python  file to the commit. 
Please have a look now.